packages/manager/apps/telecom/src/app/app.controller.js

```diff
@@ -26,6 +26,13 @@ export default class TelecomAppCtrl {
     this.ovhFeatureFlipping = ovhFeatureFlipping;
 
     this.chatbotEnabled = false;
+    this.SYSTRON_FEEDBACK_INFO = {
+      part1:
+        'The text in this manager section is provided by Systran machine translation. Share your feedback',
+      part2: 'here',
```

Comment:
For accessibility concern, I would recommend to use this content:
```suggestion
      part1:
        'The text in this manager section is provided by Systran machine translation',
      part2: 'Share your feedback.',
```